bukkit/src/main/java/co/aikar/commands/BukkitCommandContexts.java

```diff
@@ -241,6 +241,20 @@ OnlinePlayer getOnlinePlayer(BukkitCommandIssuer issuer, String lookup, boolean
         Player player = ACFBukkitUtil.findPlayerSmart(issuer, lookup);
         //noinspection Duplicates
         if (player == null) {
+            if (issuer.getManager() instanceof BukkitCommandManager) {
+                BukkitCommandManager manager = (BukkitCommandManager) issuer.getManager();
+                //Check if mc version is higher than 1.13.1 (added in 1.13.2)
+                if ((manager.mcMinorVersion > 13) || (manager.mcMinorVersion == 13 && manager.mcPatchVersion > 1)) {
+                    try {
+                        Entity entity = Bukkit.selectEntities(issuer.getIssuer(), lookup).get(0);
```

Comment:
rather than blind look up 0 relying on errors, iterate and return on first player match instead, and lack of finding should fall through to the lines below.

And we can only run this selectEntities if the input starts with @